hardware/raspberrypi/spi/README.md

```diff
@@ -136,6 +151,23 @@ When asking for say 24 MHz, the actual speed will be 15.6 MHz.
 
 Forum post: [SPI has more speeds](https://www.raspberrypi.org/forums/viewtopic.php?f=44&t=43442&p=347073)
 
```

Comment:
Should content of that thread be inserted here?

Reply:
No - I deleted it because the driver is now correct.